Update `peerDependencies` to allow latest `esbuild`

`esbuild` `0.15` has been released and this error occurs during `npm install`. `--force` is needed to ignore the error.

```sh
npm ERR! Found: esbuild@0.15.5
npm ERR! node_modules/esbuild
npm ERR!   dev esbuild@"^0.15.5" from the root project
npm ERR! 
npm ERR! Could not resolve dependency:
npm ERR! peer esbuild@"0.12 - 0.14" from esbuild-node-externals@1.4.1
npm ERR! node_modules/esbuild-node-externals
npm ERR!   dev esbuild-node-externals@"^1.4.1" from the root project
```

--- a/esbuild-node-externals/package.json
+++ b/esbuild-node-externals/package.json
@@ -21,7 +21,7 @@
     "src"
   ],
   "peerDependencies": {
-    "esbuild": "0.12 - 0.14"
+    "esbuild": "0.12 - 0.15"
   },
   "devDependencies": {
     "@types/node": "16.11.7",

